File: examples/kotlin/basic.kt
import dev.liblogit.KotlinLogit
import dev.liblogit.LibLogItK

fun main() {
    val LogIT = KotlinLogit()
    LogIT.localPath = "logs/kotlin-app.log"
    LogIT.level = "debug"

    (LogIT.at("info") shl "Kotlin app started").commit()

    val configured = LibLogItK.loadLogits("examples/config/v2-basic.json")
    (configured.getValue("AppLog").at("info").append("configured Kotlin log")).commit()
}

File: examples/javascript/basic.js
"use strict";

const { LOGIT, loadLogits, levels } = require("../../languages/javascript/src/liblogit");

const LogIT = new LOGIT();
LogIT.localPath = "logs/javascript-app.log";
LogIT.level = levels.DEBUG;

LogIT.at(levels.INFO).append("JavaScript app started").commit();

const configured = loadLogits("examples/config/v2-basic.json");
configured.AppLog.at(levels.INFO).append("configured JavaScript log").commit();
